rules/cloud/okta/okta_fastpass_phishing_detection.yml

```
@@ -0,0 +1,25 @@
+title: Okta FastPass Phishing Detection
+id: ee39a9f7-5a79-4b0a-9815-d36b3cf28d3e
+status: test
```

```suggestion
status: experimental
```
As it is a new sigma rule

---

Sorry it's been a little bit since I've made a PR.